src/cache.rs

```diff
@@ -53,8 +53,6 @@ pub struct Cache {
 
 impl Cache {
     /// Open the local store.
-    ///
-    /// `mail_dir` *must* be a subdirectory of the notmuch path.
```

Comment:
This is still true as far as `Cache` is concerned, but could be reworded like:

```markdown
`mail_cur_dir` *must* be a subdirectory of the notmuch root maildir.
```

Reply:
Oh I removed it because the names didn't match; somehow it never occurred to me to apply it to `mail_cur_dir` as-is. I'll take your suggestion, thanks!